src/methods/crop_extend.jl

```diff
@@ -92,15 +92,18 @@ function _crop_to(x, to; kw...)
         return _crop_to(x, ext; kw...)
     end
 end
-_crop_to(A, to::RasterStackOrArray; kw...) = _crop_to(A, dims(to); kw...)
+function _crop_to(A, to::RasterStackOrArray; kw...)
+    target_dims = get(kw, :dims, name.(dims(to)))
```

Comment:
You can just do this in the keywords directly without `get`, just add a `dims` keyword.